src/app.py

```diff
@@ -17,13 +18,32 @@
 def hello():
     return "Hello World!"
 
-@app.route("/sessions")      
-def home_page():
-    sessions = mongo.db.sessions
+@app.route("/sessions",methods=["POST"])      
+def sessions():
+    data = request.get_json()
+    session_collection = mongo.db.sessions
+    
+    new_session = Session( data["user_id"],data["first_received_at"],data["focus_score"],data["transcribed_at"],data["transcribed_speech"])
+    print(new_session.__dict__)
+    session_collection.insert(new_session.__dict__)
```

Comment:
When I ran this code I was given a warning that `insert` is deprecated. Try using `insert_one` instead